installer/kfserving/description.yaml

```diff
@@ -3,8 +3,8 @@ description: |
   Serverless Inferencing on Kubernetes
 namespace: kfserving-system
 install:
-  - type: manifest
-    location: kfserving-v0.5.1-v1beta1.yaml
+  - type: kustomize
```

Comment:
There is no wait here? How do you ensure that the install was successful?

Reply:
OK, let me add a wait script for start...